src/main/kotlin/org/rust/ide/newProject/Utils.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.newProject
+
+import com.intellij.ide.util.PsiNavigationSupport
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.openapiext.isHeadlessEnvironment
+import org.rust.cargo.toolchain.Cargo
+import org.rust.cargo.toolchain.Cargo.Companion.GeneratedFilesHolder
+
+fun Project.openFiles(files: GeneratedFilesHolder) {
+    if (!isHeadlessEnvironment) {
+        val navigation = PsiNavigationSupport.getInstance()
+        navigation.createNavigatable(this, files.manifest, -1).navigate(false)
+        for (file in files.sourceFiles) {
+            navigation.createNavigatable(this, file, -1).navigate(true)
+        }
+    }
+}
```

Comment:
I get empty editor in IDEA in opened file
<img width="509" alt="Screen Shot 2020-08-03 at 19 41 55" src="https://user-images.githubusercontent.com/2539310/89207615-b168dd00-d5c3-11ea-9da6-0d251f278bd5.png">
I suppose it's because of write action from which we call this method
Looks like if you wrap body into `invokeLater`, it will help